.changeset/green-geese-float.md

```diff
@@ -0,0 +1,6 @@
+---
+"@keystonejs/fields-markdown": patch
+"@keystonejs/fields": patch
```

Comment:
```suggestion
```
There's a bug in the changesets bot that causes `fields` to be picked up when it means `fields-*`, need to keep an eye out for that.

Reply:
Cool, I was wondering about this 👍